### PR TITLE
Edit title and description

### PR DESCRIPTION
**Pull request by yahiheb**
Second attempt. Read [comment](https://github.com/zkSNACKs/WasabiDoc/pull/82#discussion_r311049462).

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
This is good, thanks.

Should we add another sentence that this specifically is the documentation of Wasabi? So something like

```
The documentation of Wasabi Wallet, an open-source, non-custodial, privacy focused, desktop Bitcoin wallet. Here you find the knowledge on why privacy matters, how to properly use Wasabi, and where to contribute to the project.
```

**Comment by yahiheb** [comment]
I think the sentence is good as is, the only thing I would add is to make `Wasabi Wallet` clickable and link to the main website `https://wasabiwallet.io/`.

**Comment by MaxHillebrand** [comment]
Can the hyperlink be done with regular markdown?
`[Wasabi Wallet](https://wasabiwallet.io)`

Where exactly is this title shown? Is it also in the link preview?

**Comment by yahiheb** [comment]
I don't know if the hyperlink can be done with regular markdown.
I think this shown in the main page of the docs website, that is why we need the word `docs` in the top left of the navbar and we should add a `link` to main website and that is why I think the sentence shouldn't be too long.

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ const customBlock = require('markdown-it-custom-block')
 const youtubeEmbed = path => `<div class="ytEmbed"><iframe src="https://www.youtube-nocookie.com/embed/${path}" frameborder="0" allow="autoplay;encrypted-media;picture-in-picture" allowfullscreen></iframe></div>`
 
 module.exports = {
-  title: "Wasabi",
-  description: "Wasabi is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop.",
+  title: "Wasabi docs",
+  description: "Wasabi Wallet is an open-source, non-custodial, privacy focused, desktop Bitcoin wallet.",
```
Comment (MaxHillebrand) [inline docs/.vuepress/config.js]: After reading this line over and over, I think it is better to keep it at 
"Wasabi Wallet is an open-source, non-custodial, privacy focused, Bitcoin wallet for desktop."
I think it is more catchy to have a trinity punchline at the beginning... 
We also use this slogan on the new website and other places in the repo.

**Comment by MaxHillebrand** [comment]
Is this description related to #137?

**Comment by yahiheb** [comment]
It is not.

**Comment by yahiheb** [comment]
@MaxHillebrand Don't you think we should say wasabi **docs** on the main page of the docs website?
DocsJPG

**Pull request closed without merging**